hack/test-cmd.sh

```diff
@@ -193,6 +193,15 @@ openshift ex config patch - --type json --patch="[{\"op\": \"replace\", \"path\"
 openshift ex config patch - --patch="{\"etcdConfig\": {\"peerAddress\": \"${API_HOST}:${ETCD_PEER_PORT}\"}}" | \
 openshift ex config patch - --patch="{\"etcdConfig\": {\"peerServingInfo\": {\"bindAddress\": \"${API_HOST}:${ETCD_PEER_PORT}\"}}}" > ${SERVER_CONFIG_DIR}/master/master-config.yaml
 
+# part of the basicresources/get test suite defined in basicresources.sh
```

Comment:
and make sure you actually ensure that no config file exists, don't just run it early and hope that KUBECONFIG is unset or missing at this point